src/server.h

```diff
@@ -3923,7 +3923,7 @@ void updateActiveReplicaMastersFromRsi(rdbSaveInfo *rsi);
 uint64_t getMvccTstamp();
 void incrementMvccTstamp();
 
-#if __GNUC__ >= 7 && !defined(NO_DEPRECATE_FREE)
+#if __GNUC__ >= 7 && !defined(NO_DEPRECATE_FREE) && !defined(ALPINE)
```

Comment:
Why is alpine using free?

Reply:
it isn't, got a compile error on the noexcept specifier, so just don't compile it for alpine

Reply:
what was the compile error?

Reply:
server.h:3934:7: error: declaration of 'void* realloc(void*, size_t) noexcept' has a different exception specifier
 3934 | void *realloc(void *ptr, size_t size) noexcept;
      |       ^~~~~~~